app/autotranslate/server/settings.js

```diff
@@ -54,4 +57,18 @@ Meteor.startup(function() {
 				_id: 'AutoTranslate_ServiceProvider', value: 'deepl-translate',
 			}],
 	});
+
+	settings.add('AutoTranslate_MicrosoftAPIKey', '', {
+		type: 'string',
+		group: 'Message',
+		section: 'AutoTranslate_Microsoft',
+		public: true,
+		i18nLabel: 'AutoTranslate_APIKey',
```

Comment:
Missing the translation on en.i18n.json

Reply:
This one? https://github.com/assistify/Rocket.Chat/blob/core/feature/ms-translate/packages/rocketchat-i18n/i18n/en.i18n.json#L446

Reply:
Yes, it's missing here

Reply:
Ah, sure, sorry.

Reply:
Sorry, @rodrigok , I don't get it. `AutoTranslate_APIKey` is in `en.i18n.json`, see the above link. `AutoTranslate_Microsoft ` is missing, I'll add that.